vendor/bat-native-ledger/src/bat/ledger/internal/media/twitter.cc

```diff
@@ -176,9 +294,175 @@ void MediaTwitter::SavePublisherInfo(
                           duration,
                           window_id,
                           callback);
+
   if (!media_key.empty()) {
     ledger_->SetMediaPublisherInfo(media_key, publisher_key);
   }
 }
 
+void MediaTwitter::OnSaveMediaVisit(
+    ledger::Result result,
+    ledger::PublisherInfoPtr info) {
+}
+
+void MediaTwitter::FetchDataFromUrl(
+    const std::string& url,
+    braveledger_media::FetchDataFromUrlCallback callback) {
+  ledger_->LoadURL(url,
+                   std::vector<std::string>(),
+                   std::string(),
+                   std::string(),
+                   ledger::URL_METHOD::GET,
+                   callback);
+}
+
+void MediaTwitter::OnMediaActivityError(const ledger::VisitData& visit_data,
+                                        uint64_t window_id) {
+  std::string url = TWITTER_TLD;
```

Comment:
minor: would be more clear to name this `domain` instead of `url`. otherwise this is confusing:

```
  new_data.domain = url;
  new_data.url = "https://" + url;
```